src/util/irep_serialization.cpp

```diff
@@ -38,13 +38,6 @@ void irep_serializationt::write_irep(
     reference_convert(it->second, out);
   }
 
-  forall_named_irep(it, irep.get_comments())
-  {
-    out.put('C');
```

Comment:
Since we no longer put this, reading a binary generated by a newer version of `goto-cc` using an older version will place all comments in the non-comment map. Thus I think a change to the goto binary version is required, but maybe I'm getting this wrong.

Reply:
No, the reading in old versions happens using .add, which will put everything into the right map.